homeworks/08-cats-catcher/src/App/generateFiled.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect, useCallback } from "react";
+import random from "lodash/random";
+
+import { WINDOW_SIZE } from "./contants";
+
+const generateGameField = (): boolean[][] =>
+  Array.from({ length: 10 }, () => Array.from({ length: 10 }, () => true));
+
+const getPositions = (field: boolean[][]): number[] => {
+  const x = random(0, 9);
+  const y = random(0, 9);
+
+  const isCellFree = field[x][y];
+
+  if (isCellFree) {
+    field[x][y] = false;
+    return [x * WINDOW_SIZE, y * WINDOW_SIZE];
+  }
+
+  return getPositions(field);
+};
+
+const generatePositions = () => {
```

Comment:
I'll try to guess, position means windows? Make sense to rename function like `generateWindows` or similar.